From 087387170dbfbf41dc84ba0f21136f4dfd80c085 Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Fri, 21 Sep 2012 10:49:39 +0900
Subject: [PATCH 14/39] Revert "base_filter: set timestamps if not set"

This reverts commit aa6e3c52d398e4e43296b628c6339af8994c6674.

This may cause trouble with raw video streams in which a frame
would be constructed of multiple NAL units.
Furthermore this could be unnecessary since lagacyh264parse
manage to give appropriate timestamp for such raw stream data.
---
 omx/gstomx_base_filter.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/omx/gstomx_base_filter.c b/omx/gstomx_base_filter.c
index 3a43455..674ad84 100644
--- a/omx/gstomx_base_filter.c
+++ b/omx/gstomx_base_filter.c
@@ -682,12 +682,6 @@ pad_chain (GstPad * pad, GstBuffer * buf)
         else
           omx_buffer->nFlags = OMX_BUFFERFLAG_ENDOFFRAME;
 
-        if ((GST_BUFFER_TIMESTAMP (buf) == 0) ||
-            (GST_BUFFER_TIMESTAMP (buf) == GST_CLOCK_TIME_NONE)) {
-          static guint64 cnt = 0;
-          omx_buffer->nTimeStamp = cnt++;
-        }
-
         GST_LOG_OBJECT (self, "release_buffer");
                 /** @todo untaint buffer */
         g_omx_port_release_buffer (in_port, omx_buffer);
-- 
1.7.9.5

